patients/models.go

```diff
@@ -170,4 +170,27 @@ func (u UpdateDescription) applyUpdatesToExistingProfile(profile map[string]inte
 
 type UpdatedFields struct {
 	Patient
+
+	// Partial updates to nested fields are encoded using dot notation in CDC events
+	ProviderConnectionRequestsDexcom ConnectionRequests `bson:"providerConnectionRequests.dexcom"`
+	ProviderConnectionRequestsTwiist ConnectionRequests `bson:"providerConnectionRequests.twiist"`
+}
+
+func (u UpdatedFields) GetUpdatedConnectionRequests() ConnectionRequests {
+	var requests ConnectionRequests
+	if u.ProviderConnectionRequests != nil {
+		for _, r := range u.ProviderConnectionRequests {
+			for _, v := range  r {
+				requests = append(requests, v)
+			}
+		}
+	}
+	for _, v := range  u.ProviderConnectionRequestsDexcom {
+		requests = append(requests, v)
+	}
+	for _, v := range  u.ProviderConnectionRequestsTwiist {
+		requests = append(requests, v)
```

Comment:
Would it be simpler to just do a `requests = append(requests, u.ProviderConnectionRequestsTwiist...)` and the same for `ProviderConnectionRequestsDexcom` ?